data/json/recipes/chem/fuel.json

```diff
@@ -0,0 +1,128 @@
+[
+  {
+    "result": "chem_ethanol",
+    "type" : "recipe",
+    "byproducts": [ "water_clean" ],
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "brewing_cookbook", 4 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 2, "amount": 1 },
+        { "id": "BREW", "level": 2, "amount": 1 }
+     ],
+     "tools": [
+       [ [ "fire", -1 ], [ "hotplate", 30 ], [ "toolset", 30 ] ],
+       [ [ "thermometer", -1 ] ]
+     ],
+     "components": [ [
+       [ "vodka", 14 ],
+       [ "whiskey", 14 ],
+       [ "single_malt_whiskey", 14 ],
+       [ "brandy", 14 ],
+       [ "rum", 14 ],
+       [ "gin", 14 ],
+       [ "tequila", 14 ],
+       [ "bum_wine", 56 ],
+       [ "fruit_wine", 56 ],
+       [ "moonshine", 14 ],
+       [ "mixed_alcohol_strong", 14 ]
+    ] ]
+  },
+  {
+    "result": "chem_ethanol",
+    "type" : "recipe",
+    "id_suffix": "redistilled",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": 30000,
+    "book_learn": [ [ "textbook_chemistry", 5 ], [ "brewing_cookbook", 4 ] ],
+    "qualities": [
+        { "id": "BOIL", "level": 2, "amount": 1 },
+        { "id": "BREW", "level": 2, "amount": 1 }
+     ],
+     "tools": [
+       [ [ "fire", -1 ], [ "hotplate", 30 ], [ "toolset", 30 ] ],
+       [ [ "thermometer", -1 ] ]
+     ],
+     "components": [ [ [ "denat_alcohol", 250 ] ] ]
```

Comment:
From what I can find, this isn't possible. Ethanol and methanol aren't separable with non-lab, non-industrial tools.